test/common.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs');
+
+const REPORT_SECTIONS = [
+  'NodeReport',
+  'JavaScript Stack Trace',
+  'JavaScript Heap',
+  'System Information'
+];
+
+exports.findReports = (pid) => {
+  // Default NodeReport filenames are of the form NodeReport.<date>.<time>.<pid>.<seq>.txt
+  const filePattern = new RegExp('^NodeReport\\.\\d+\\.\\d+\\.' + pid + '\\.\\d+\\.txt$');
+  const files = fs.readdirSync(".");
+  return files.filter((file) => filePattern.test(file));
+};
+
+exports.validate = (t, report, pid) => {
+  t.test('Validating ' + report, (t) => {
+    fs.readFile(report, (err, data) => {
+      const reportContents = data.toString();
+      const plan = REPORT_SECTIONS.length + (pid ? 1 : 0);
+      t.plan(plan);
+      if (pid) {
+        t.match(reportContents, new RegExp('Process ID: ' + pid), 'Checking report contains expected process ID ' + pid);
```

Comment:
Line to long. We need to add eslint to this repo, but in meantime, make it easy on whoever does that and limit to 80 columns.